aiorussound/models.py

```diff
@@ -6,6 +6,18 @@
 from mashumaro.mixins.orjson import DataClassORJSONMixin
 
 
+@dataclass
+class RussoundFavorite:
+    """Russound Favorite."""
+
+    favorite_id: int
+    issystemfavorite: bool
```

Comment:
Let's use snake case for compound words, so `is_system_favorite` instead of issystemfavorite. Please do so for the other attributes as well.